.gitignore

```diff
@@ -65,3 +65,7 @@ examples/common/java/*/bin/
 .vscode/
 .DS_Store
 
+### Ignore Pytest Automation Test Results ###
+tests/validaiton_suite/results/*
+tests/validaiton_suite/report.json
+tests/validaiton_suite/pytest-logs.txt
```

Comment:
Keep a separate .gitignore file under tests/validation_suite.  Then these changes are not required